sched/scheduler/jobStatus.go

```diff
@@ -0,0 +1,83 @@
+package scheduler
+
+import (
+	s "github.com/scootdev/scoot/saga"
+	"github.com/scootdev/scoot/sched"
+)
+
+type JobStatus struct {
+	Id         string
+	Status     sched.Status
```

Comment:
If you have a suggestion that differentiates this happy to update, but best I could come up with other than JobStatusObject